packages/lib-classifier/src/plugins/tasks/DrawingTask/components/tools/DrawingToolRoot.js

```diff
@@ -1,56 +1,100 @@
 import PropTypes from 'prop-types'
-import React, { useState } from 'react'
+import React, { forwardRef, useState } from 'react'
+import styled from 'styled-components'
+import draggable from '../components/draggable'
 
-const STROKE_WIDTH = 1.5
-const SELECTED_STROKE_WIDTH = 2.5
+const STROKE_WIDTH = 2
+const SELECTED_STROKE_WIDTH = 3
 
-function DrawingToolRoot ({ children, tool }) {
-  const [ active, setActive ] = useState(true)
+const StyledGroup = styled('g')`
+  :focus {
+    outline: 1px solid black;
```

Comment:
The focus color used in the theme is `#addde0`. How does it look using this? Perhaps you can use the `withTheme` decorator instead to get the theme colors?

Reply:
This is a debugging line.